pkg/apis/core/validation/validation.go

```diff
@@ -1047,6 +1047,7 @@ func validateDownwardAPIVolumeFile(file *core.DownwardAPIVolumeFile, fldPath *fi
 		if file.ResourceFieldRef != nil {
```

Comment:
are we checking if status.hostIPs are valid anywhere?

Reply:
Good catch

Reply:
That said, I also don't see status.hostIP getting validated anywhere :-/

We should avoid making the problem worse, though. We may need to do something like https://github.com/kubernetes/kubernetes/commit/23e9fb1bb552bd0098770ad26bb3d5f60f8f2c06#diff-c713e8919642d873fdf48fe8fb6d43e5cb2f53fd601066ff53580ea655948f0dR3862